environment.yml

```
@@ -9,6 +9,7 @@ dependencies:
   - python=3.9
   - pip=22
   # Optional dependencies
+  - panel=0.12.4
```

Somehow I had to update to panel=0.13.0 for the interactive plot to show up. Does it work for you with panel=0.12.4?
```suggestion
  - panel=0.13.0
```